app/resources/results.html.template

```diff
@@ -46,16 +46,20 @@
       // third party search API directly here, to workaround AJAX's same
       // origin policy, and for authorization.
       function perform_third_party_search(id) {
-        jQuery.ajax({
-            url: repo_url + '/third_party_search',
-            data: {
-                query: query,
-                lang: lang,
-                query_type: query_type,
-                search_engine_id: id
-            },
-            dataType: 'json'
-        }).done(function(r) {
+        fetch(repo_url + '/third_party_search', {
```

Comment:
Have you checked third party search works fine after your change, just in case? If not, I can probably help you to test it since it's a bit tricky to set up an environment to test it...

Reply:
I tested the frontend changes, but I did it by hacking the backend to return hard-coded values. Is it easy to find a query that will return results (once you configure it to point at the real third-party) or do you test it some other way?

Reply:
That would probably be sufficient, thanks! "安否　一郎" is a sample query which always returns the result, so we can try the query after pushing this change in production.